### PR TITLE
Aptiv Integration (map-supplier-loader)

### PR DESCRIPTION
**Pull request by vrajur-nd**
PR integrates current visualization with aptiv-specific functionality and features:
 - reading of multiple files and `visualizationMode` passed in from VeritasApplication
 - updated `laneLoader.js` to handle loading of map-supplier lanes as well as comparison lanes (also can handle loading larger sized flatbuffer files)
 - sidebar display for map supplier and comparison lanes
 - updated aptiv-specific default values and visualization settings
 

Functionality tested against `VeritasApplication` - [338184f](https://github.com/NextDroid/VeritasApplication/commit/338184f11c802c50973d5124030855defbb5d68c) on both aptiv and non-aptiv datasets

### CODE REVIEW

File: demo/radar.html [hunk 1]
```diff
@@ -606,13 +626,16 @@
 		}
 
 
-// TODO Just for testing:
-// tweenInitAll();
+		// TODO Just for testing:
```
Comment (calistenson) [inline demo/radar.html]: Are these two lines necessary for testing each time?

File: demo/radar.html [hunk 2]
```diff
@@ -625,6 +648,63 @@
...
+					viewer.scene.dispatchEvent({
+						"type": "truth_layer_added",
+						"truthLayer": lanesLayer
+					});
+				});
+			}
+
+
 		}
 // TODO TILL HERE
```
Comment (calistenson) [inline demo/radar.html]: What does this last TODO refer to?

File: common/custom-sidebar.js [hunk 3]
```diff
@@ -89,13 +90,28 @@ export function updateSidebar() {
...
-  window.viewer.scene.addEventListener("sensor_layer_added", onSensorLayerAdded);
   window.viewer.scene.addEventListener("assessments_layer_added", onAssessmentsLayerAdded);
 
 
+  if (vizConfiguration == "aptivLanes") {
+    const HdMapProvidersTree = tree.jstree('create_node', "#", { "text": "<b>HD Map Providers</b>", "id": "HdMapProvidersViz"}, "last", false, false);
+    tree.jstree("check_node", HdMapProvidersTree);
+    let onMapProviderLayerAdded = (e) => {
+      let mapProviderLayer = e.mapLayer;
+      let mapIcon = `${Potree.resourcePath}/icons/focus.svg`; // TODO Fix this
```
[the remark this answers is not in the document]
Comment (vrajur-nd) [reply common/custom-sidebar.js]: Updated - this is just to remind me to find a better icon to use

File: demo/radar.html (hunk 1 above)
Comment (vrajur-nd) [reply demo/radar.html]: Removed

File: demo/radar.html (hunk 2 above)
Comment (vrajur-nd) [reply demo/radar.html]: Removed

File: demo/radar.html [hunk 4]
```diff
@@ -625,11 +644,68 @@
...
+							} catch (e) {
+								console.log(`Couldn't load ${filename}: ${e}`);
+							}
+						}
+					}
+				}
+			}
+
+			// Load Comparison Dataset (hardcoded to one for now)
+			debugger;
```
Comment (calistenson) [inline demo/radar.html]: @vrajur-nd can you remove this when you get a chance?

Comment (vrajur-nd) [reply demo/radar.html]: branch was already deleted so addressed in separate PR

**Pull request merged**